### PR TITLE
Add info about `plangenerator` to README.

### PR DESCRIPTION
**Pull request by anvial**
## Description

This PR adds info about the `testing/plangenerator` functionality, which helps to create Acceptance tests for README.

Fixes: 

## Type of change

- Requires a documentation update

## Environment

- Juju controller version: any

- Terraform version: any

## QA steps

```
cat README.md
```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,6 +85,18 @@ Prior to running the tests locally, ensure you have the following environmental
...
 
+### Writing Acceptance Tests
+
+Acceptance tests usually require Terraform plans to be included to the various steps of the test. 
+This plans can be generated with help of `testing/plangenerator` package. You may use `GetStringFromTemplateWithData`
+function to generate a plan from a template. This function will replace the placeholders in the template with the data. 
+
+The example can be found in the comments to `GetStringFromTemplateWithData` function.
+
+### Testing
```
Comment (cderici) [inline README.md:96]:
```suggestion
### Running Tests
```

Comment (anvial) [reply README.md:96]: thx

File: README.md [hunk 2]
```diff
@@ -85,6 +85,18 @@ Prior to running the tests locally, ensure you have the following environmental
 - `JUJU_PASSWORD`
 - `JUJU_CA_CERT`
 
+### Writing Acceptance Tests
```
[the remark this answers is not in the document]
Comment (anvial) [reply README.md:88]: agreed

**Review by cderici — APPROVED** [review]
Yeah this LGTM 👍

**Pull request merged**